src/Components/SpreadModals/EventsModal.js

```diff
@@ -0,0 +1,56 @@
+import EventsType from '../../Constants/EventsType.js';
+
+export default function EventsModal({ setShowModal, monthEvents }) {
+  return (
+    <>
```

Comment:
No need for the fragment here

Reply:
i need it here cz i have two divs one for the modal and one for the gray background